src/model/enums/misc.rs

```diff
@@ -91,3 +93,21 @@ pub enum Modality {
     Major = 1,
     NoResult = -1,
 }
+
+/// Limit the response to a particular market
+///
+/// FromToken is the same thing as setting the market parameter to the user's country.
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
+pub enum Market {
+    Country(Country),
+    FromToken,
+}
+
+impl Into<String> for Market {
```

Comment:
Why not use `ToString` instead of `Into<String>` by the way? That way you can use `market.to_string`, which:

* Avoids type inference issues (I saw a case where you had to declare `market: String` to let Rust know it's `Into::<String>`)
* `to_string` is more explicit and easier to understand for the reader
* The struct is borrowed instead of moved, so you can keep using it

Reply:
Good question, guess I didn't really think about it. I'll fix it.